Source/Dafny/Compiler-Csharp.cs

```diff
@@ -2356,7 +2356,7 @@ private class CSharpCompilationResult
       };
 
       if (DafnyOptions.O.Optimize) {
-        cp.CompilerOptions += " /optimize /define:DAFNY_USE_SYSTEM_COLLECTIONS_IMMUTABLE";
+        cp.CompilerOptions += " /optimize ";
         cp.CompilerOptions += " /lib:" + crx.libPath;
         foreach (var filename in crx.immutableDllFileNames) {
```

Comment:
Note "System.Collections.Immutable.dll" is in this list and needs to be added unconditionally now.

Reply:
Thanks!